source/speech/speech.py

```diff
@@ -1861,6 +1860,105 @@ def getPropertiesSpeech(  # noqa: C901
 	return textList
 
 
+def speakKeyboardShortcuts(keyboardShortcutsStr: Optional[str]) -> SpeechSequence:
+	speak(getKeyboardShortcutsSpeech(keyboardShortcutsStr))

```

Comment:
should this return here based on the return type?

```suggestion
	return speak(getKeyboardShortcutsSpeech(keyboardShortcutsStr))
```

Reply:
No, the return type was wrong. I have fixed it in 449e876.